LL/senchas_síl_ébir.ttl

```diff
@@ -655,7 +655,7 @@
     rel:descendantOf <#Dare>.
 
 <#MacCon>
-    a foaf:Person;
+    a irishTitles:Rí;
```

Comment:
Looking at the text again, I am not so sure that Mac Con is being described as a king. Do you want to have a look yourself? LL, Book VI, Section 5, p.1374, first para.